src/program/lwaftr/tests/selftest.sh

```diff
@@ -13,6 +13,15 @@ else
     TEST_WHAT="*"
 fi
 
+# Create a directory for JIT migrated configs so that each configuration file
```

Comment:
Can we remove this and all the bits about generating files with the specific PCI addresses in them?